### PR TITLE
FAT 839 autotest loans claim returned is added

### PR DESCRIPTION
**Pull request by penfriend**

### CODE REVIEW

File: cypress/integration/loans/loans-claim-returned.spec.js [hunk 1]
```diff
@@ -0,0 +1,177 @@
+import moment from 'moment';
+import uuid from 'uuid';
+import devTeams from '../../support/dictionary/devTeams';
+import permissions from '../../support/dictionary/permissions';
+import TopMenu from '../../support/fragments/topMenu';
+import TestTypes from '../../support/dictionary/testTypes';
+import CheckInActions from '../../support/fragments/check-in-actions/checkInActions';
```
Comment (MaksimTerekhov) [inline cypress/integration/loans/loans-claim-returned.spec.js:7]: 'CheckInActions' is defined but never used

File: cypress/integration/loans/loans-claim-returned.spec.js [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+import ServicePoints from '../../support/fragments/settings/tenant/servicePoints/servicePoints';
+import UserEdit from '../../support/fragments/users/userEdit';
+import generateItemBarcode from '../../support/utils/generateItemBarcode';
+import getRandomPostfix from '../../support/utils/stringTools';
+import AppPaths from '../../support/fragments/app-paths';
```
Comment (MaksimTerekhov) [inline cypress/integration/loans/loans-claim-returned.spec.js:12]: 'AppPaths' is defined but never used

File: cypress/integration/loans/loans-claim-returned.spec.js [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+import Loans from '../../support/fragments/loans/loansPage';
+import InventoryInstance from '../../support/fragments/inventory/inventoryInstance';
```
Comment (MaksimTerekhov) [inline cypress/integration/loans/loans-claim-returned.spec.js:14]: 'InventoryInstance' is defined but never used

File: cypress/integration/loans/loans-claim-returned.spec.js [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+import InventoryInstances from '../../support/fragments/inventory/inventoryInstances';
+import UsersOwners from '../../support/fragments/settings/users/usersOwners';
+import UserLoans from '../../support/fragments/users/loans/userLoans';
+import ItemVeiw from '../../support/fragments/inventory/inventoryItem/itemVeiw';
```
Comment (MaksimTerekhov) [inline cypress/integration/loans/loans-claim-returned.spec.js:18]: 'ItemVeiw' is defined but never used.

File: cypress/integration/loans/loans-claim-returned.spec.js [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+import ConfirmItemStatusModal from '../../support/fragments/users/loans/confirmItemStatusModal';
+
+import Users from '../../support/fragments/users/users';
```
Comment (MaksimTerekhov) [inline cypress/integration/loans/loans-claim-returned.spec.js:21]: 'Users' is defined but never used.

File: cypress/integration/loans/loans-claim-returned.spec.js [hunk 6]
```diff
@@ -0,0 +1,177 @@
...
+    UsersSearchPane.searchByKeywords(username);
+    UsersSearchPane.openUser(username);
+    UsersCard.openLoans();
+    UsersCard.showOpenedLoans();
+  };
+
+  const openLoanWithParametrizedItemStatus = (itemStatus) => {
+    openOpenedUserLoans(this.userData);
+    UserLoans.selectLoan(itemStatus);
+    // cy.do(rowInList.find(HTML(including(barcode))).click());
```
Comment (MaksimTerekhov) [inline cypress/integration/loans/loans-claim-returned.spec.js:68]: remove comment pls

File: cypress/integration/loans/loans-claim-returned.spec.js [hunk 7]
```diff
@@ -0,0 +1,177 @@
...
+              UserLoans.declareLoanLost();
+              ConfirmItemStatusModal.confirmItemStatus();
+            // ItemVeiw.verifyItemStatus('DeclaredToLost');
+            });
+          }
+        });
+      });
+  });
+
+  // after('Delete New Service point, Item and User', () => {
```
Comment (MaksimTerekhov) [inline cypress/integration/loans/loans-claim-returned.spec.js:143]: delete comments pls

File: cypress/integration/loans/loans-claim-returned.spec.js [hunk 8]
```diff
@@ -0,0 +1,177 @@
...
+            loanDate: moment.utc().format(),
+            servicePointId: servicePoint.id,
+            userBarcode: userData.barcode,
+          });
+          if (index % 2 === 0) {
+            cy.loginAsAdmin().then(() => {
+              openOpenedUserLoans(userData);
+              UserLoans.declareLoanLost();
+              ConfirmItemStatusModal.confirmItemStatus();
+            // ItemVeiw.verifyItemStatus('DeclaredToLost');
```
Comment (MaksimTerekhov) [inline cypress/integration/loans/loans-claim-returned.spec.js:136]: comment

File: cypress/integration/loans/loans-claim-returned.spec.js (hunk 1 above)
Comment (penfriend) [reply cypress/integration/loans/loans-claim-returned.spec.js:7]: Done.

File: cypress/integration/loans/loans-claim-returned.spec.js (hunk 2 above)
Comment (penfriend) [reply cypress/integration/loans/loans-claim-returned.spec.js:12]: Done.

File: cypress/integration/loans/loans-claim-returned.spec.js (hunk 3 above)
Comment (penfriend) [reply cypress/integration/loans/loans-claim-returned.spec.js:14]: Done.

File: cypress/integration/loans/loans-claim-returned.spec.js (hunk 4 above)
Comment (penfriend) [reply cypress/integration/loans/loans-claim-returned.spec.js]: Done.

File: cypress/integration/loans/loans-claim-returned.spec.js (hunk 5 above)
Comment (penfriend) [reply cypress/integration/loans/loans-claim-returned.spec.js:21]: Done.

File: cypress/integration/loans/loans-claim-returned.spec.js (hunk 6 above)
Comment (penfriend) [reply cypress/integration/loans/loans-claim-returned.spec.js]: Done.

File: cypress/integration/loans/loans-claim-returned.spec.js (hunk 7 above)
Comment (penfriend) [reply cypress/integration/loans/loans-claim-returned.spec.js]: Done

File: cypress/integration/loans/loans-claim-returned.spec.js (hunk 8 above)
Comment (penfriend) [reply cypress/integration/loans/loans-claim-returned.spec.js]: Done.

**Review by MaksimTerekhov — APPROVED** [review]

**Review by TetianaParanich — APPROVED** [review]

**Pull request merged**